src/components/Home.js

```diff
@@ -0,0 +1,15 @@
+import Book from './Book';
+
+const Home = () => (
+  <div className="home">
+    <h1>Welcome to Bookstore</h1>
+    <Book title="river-between" author="ngugi" id={1} />
+    <form>
+      <input type="text" placeholder="title" />
+      <input type="text" placeholder="author" />
+      <button type="submit">Add Book</button>
+    </form>
```

Comment:
**[OPTIONAL]**

- [ ] It would be very great if you add this form in a different component to make your component reusable 👍 and also it will allow us to organize our code of form 😊

Reply:
I like this suggestion 😄.  Having form on different component will make this code more organized, Thanks for reminder 💯